data/remnant.txt

```diff
@@ -168,3 +168,221 @@ outfitter "Remnant"
 	"Fuel Pod"
 	"Cargo Scanner"
 	"Outfit Scanner"
+
+
+# Hails:
+phrase "uncontacted remnant hostile hail"
+	word
+		"Your "
+	word
+		"ship"
+		"technology"
+		"weapons"
+		"cargo"
+	word
+		" "
+	word
+		"will"
+		"might"
+	word
+		"be of great help to us"
+		"help us plunder more raiders like you"
+		"let us live for a very long time"
+		"be helpful for our future generations"
+		"make us more independent from unreliable human technology"
+	word
+		". "
+	word
+		"Surrender"
+		"Desist"
+		"Stop"
+	word
+		", "
+	word
+		"before your ship becomes space debris"
+		"before we kill you"
+		"before you see your possession is no more"
+		"before your thing is no longer yours"
+	word
+		"."
+
+phrase "uncontacted remnant hostile hail"
+	word
+		"Raiders"
+		"Pirates"
+		"Plunderers"
+		"Stealers"
+	word
+		" "
+	word
+		"like you "
+	word
+		"will be "
+		"are "
+	word
+		"killed"
+		"terminated"
+		"plundered as well"
+	word
+		". "
+	word
+		"You "
+	word
+		"have no chance"
+		"have little chance"
+	word
+		"against us"
+		"on our worlds"
+		"on our planets"
+		"on Ember Waste"
+		"on this region of the galaxy"
+	word
+		", so "
+	word
+		"be gone"
+		"leave us alone"
+		"leave this region"
+		"don't dare to attack us"
+	word
+		"."
+		"!"
+
+phrase "angered remnant hail"
+	word
+		"Please"
+		"You must"
+	word
+		" "
+	word
+		"surrender"
+		"desist"
+		"desist and surrender"
+		"surrender yourself"
+		"desist and surrender yourself"
+	word
+		", "
+	word
+		"alien"
+		"unknown individual"
+	word
+		". "
+	word
+		"Please do not "
+		"Do not "
+	word
+		"be afraid"
+	word
+		" "
+	word
+		"of us"
+		"of the Remnant"	
+
+phrase "angered remnant hostile hail"
+	word
+		"We"
+		"The Remnant"
+		"Our ship"
+	word
+		"will"
+		"is going to"
+	word
+		" "
+	word
+		"stop"
+		"cease"
+	word
+		" "
+	word
+		"following"
+		"intercepting"
+		"attacking"
+	word
+		" "
+	word
+		"you"
+		"your ship"
+	word
+		"if you will"
+	word
+		"land and take the test"
+		"surrender"
+		"stop escaping from us"
+		"cease escaping from us"
+	word
+		"."
+
+phrase "contacted remnant hostile hail"
+	word
+		"You "
+	word
+		"human"
+		"pure one"
+		"unaltered"
+	word
+		"are just like"
+		"have similarities with"
+		"are the same as"
+	word
+		" the "
+	word
+		"Alphas"
+		"other pirates"
+		"alien raiders"
+		"other raiders"
+		"other plunderers"
+	word
+		". "
+	word
+		"You"
+	word
+		" "
+	word
+		"should be"
+		"must be"
+	word
+		"destroyed"
+		"killed"
+		"out of our space"
+		"gone"
+
+phrase "contacted remnant hostile disabled hail"
+	word
+		"Human"
+		"Brother"
```

Comment:
initial comment was hidden in a "show outdated", even though it's not, so I'll repeat:
the game never assumes the player gender, while this assumes male, take this word out